### PR TITLE
ecs fix and unification in monorepo

### PR DESCRIPTION
**Pull request by boris-brtan**
| Q             | A
| ------------- | ---
|Description, reason for the PR| problems with ObjectIsCreatedByFactorySniff in packages
|New feature| No 
|BC breaks| Yes 
|Fixes issues| 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

File: packages/coding-standards/README.md [hunk 1]
```diff
@@ -25,15 +25,14 @@ php composer require shopsys/coding-standards
 
 ## Usage
 
-Create `custom-coding-standard.yml` config file in your project which includes predefined rulesets. 
+Create `easy-coding-standard.yml` config file in your project which includes predefined ruleset. 
 You can also [customize the rules](./UPGRADE.md)
```
Comment (PetrHeinz) [inline packages/coding-standards/README.md]: There is no shopsys/coding-standards/UPGRADE.md

File: packages/framework/easy-coding-standard.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 imports:
-    - { resource: '../**/coding-standards/easy-coding-standard.yml' }
+    - { resource: '%vendor_dir%/shopsys/coding-standards/easy-coding-standard.yml', ignore_errors: true  }
```
Comment (PetrHeinz) [inline packages/framework/easy-coding-standard.yml]: The reason to use `ignore_errors` should be explaind in the commit msg imho

File: packages/product-feed-zbozi/easy-coding-standard.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+imports:
+    - { resource: '%vendor_dir%/shopsys/coding-standards/easy-coding-standard.yml', ignore_errors: true  }
+
+parameters:
+    skip:
+        ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+            - '*/src/ZboziPluginDataFixture.php'
+            - '*/tests/Unit/ZboziFeedTest.php'
+        Shopsys\CodingStandards\Sniffs\ObjectIsCreatedByFactorySniff:
+          - '*/tests/*'
```
Comment (PetrHeinz) [inline packages/product-feed-zbozi/easy-coding-standard.yml]: In some of the config files, there is a new line missing at the end. This results in diffs with worse readability, but it's not that important to fix those - you can leave it as-is.

File: build.xml [hunk 4]
```diff
@@ -70,7 +69,7 @@
         </exec>
         <exec executable="${path.ecs.executable}" logoutput="true" passthru="true" checkreturn="true">
             <arg value="check" />
```
Comment (PetrHeinz) [inline build.xml]: The flag `--verbose` is not used here, but it is used in other build definitions. Can you explain or fix it?

File: build.xml [hunk 5]
```diff
@@ -98,8 +97,8 @@
     <target name="standards-fix-packages" description="Automatically fixes some coding standards violations in all packages.">
         <exec executable="${path.ecs.executable}" logoutput="true" passthru="true" checkreturn="true">
             <arg value="check" />
```
Comment (PetrHeinz) [inline build.xml]: The flag `--verbose` is not used here, but it is used in other build definitions. Can you explain or fix it?

File: build.xml [hunk 6]
```diff
@@ -70,7 +69,7 @@
         </exec>
         <exec executable="${path.ecs.executable}" logoutput="true" passthru="true" checkreturn="true">
             <arg value="check" />
-            <arg value="--config=${path.ecs.configuration}"/>
+            <arg value="--clear-cache" />
```
Comment (PetrHeinz) [inline build.xml]: Clearing the cache with every execution of `standards-packages` might be a bit painful as there is no `standards-packages-diff` variant and it is called during `build-demo-dev`.

Please reconsider this behavior.

File: build.xml (hunk 4 above)
Comment (boris-brtan) [reply build.xml]: verbose was added

File: build.xml (hunk 5 above)
Comment (boris-brtan) [reply build.xml]: verbose was added

File: build.xml (hunk 6 above)
Comment (boris-brtan) [reply build.xml]: clearing of cache was removed

**Comment by TomasVotruba** [comment]
I think there is one missed spot: https://github.com/shopsys/shopsys/blob/master/packages/http-smoke-testing/.travis.yml

Or is that intentional inconsistency?

**Pull request merged**